ibu-imager/seedcreator/seedcreator.go

```diff
@@ -203,6 +216,52 @@ func (s *SeedCreator) createContainerList() error {
 	return nil
 }
 
+func (s *SeedCreator) backupSeedCertificates() error {
+	s.log.Info("Backing up seed cluster certificates for recert tool.")
+	if err := os.MkdirAll(backupCertsDir, os.ModePerm); err != nil {
+		return err
+	}
+
+	args := []string{"extract", "-n", "openshift-config", "configmap/admin-kubeconfig-client-ca",
```

Comment:
I think it might be cleaner to use `net/http` library to get these resources from the kube-apiserver, which was meant for more programmatic access, rather than relying on `oc` which is a human interface. Mostly in terms of stability, I think the latter is much more likely to change with time, but also we have much better error handling and understanding of `net/http` than weird exit codes we might get from `oc`

Reply:
Use the same client we use in seedCreator object

Reply:
Ah didn't realize we already have the client as a dependency. Definitely use the client then

Reply:
k, thanks for the pointer, this is now using the suggested client.